python-package/pyproject.toml

```
@@ -154,6 +143,7 @@ select = [
     "D",
     # pycodestyle
     "E",
+    "W",
```

Add [pycodestyle Warnings](https://docs.astral.sh/ruff/rules/#warning-w)

---

```suggestion
    # pycodestyle (warnings)
    "W",
```

Sorry, missed this before I approved.

Could you:

* add a comment like this one
* change the comment over `"E"` `# pycodestyle (errors)`
* keep this list alphabetically organized by rule code (so move this `"W"` to the end, and put `isort` after `"F"`

I think that makes it a bit easier to find the relevant entry in the config and map it to the type of check. 

Very very minor suggestion, so if you disagree at all then just merge this.

---

Sure, will do.

---

Done in 4cb519e76857a0a2f261948034d0ab098e63129b. Is it OK now?